docs/guides/Redux-Integration.md

```diff
@@ -85,3 +85,35 @@ To make jest tests work with your react-navigation app, you need to change the j
   ]
 }
 ```
+
+## Extra : Handling back button in android
+
+By using the following snippet, your nav component will be aware of the back button press actions and will correctly interact with your stack. This is really useful on Android.
+
+```javascript
+class ReduxNavigation extends React.Component {
```

Comment:
Is this example exclusive to redux integration? Can this code snippet be used to modify the behavior of the Android back button without redux integration? If so, is there another place in the docs that would be better suited for this example?

Reply:
I think, for now, this code example is in the right place. I don't believe you can override the existing back handler _unless_ you manage nav state yourself.

Reply:
@matthamil I think the same way as @spencercarli, if it's possible i'm not aware of it and I would be happy to know it :)